assets/js/modules/analytics/setup.js

```diff
@@ -259,6 +259,12 @@ class AnalyticsSetup extends Component {
 
 			const responseData = await data.get( 'modules', 'analytics', 'get-accounts', queryArgs, false );
 			if ( 0 === responseData.accounts.length ) {
+				newState = {
+					...newState,
+					errorCode: true,
```

Comment:
This should be an actual error code, not a boolean. You could for example use `no_account`.